contact.html

```diff
@@ -0,0 +1,613 @@
+                                            <div data-cmp-is="adaptiveFormText" id="text-53cc58dc93" name="text_demo" data-cmp-adaptiveformcontainer-path="/content/core-components-examples/library/adaptive-form/text/jcr:content/root/responsivegrid/demo/component/guideContainer" class="cmp-adaptiveform-text">
+                                              <div tabindex="0">
+                                                    <p>help@leaderimpact.com</p>
+                                              </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="container responsivegrid button-text aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--6 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                                        <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                            <a class="cmp-button" type="button">
+                                                <span class="cmp-button__text">
+                                                    Email Us
+                                                </span>
+                                            </a>
+                                        </div>
+                                        <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                            <a class="cmp-button" type="button">
+                                                <span class="cmp-button__text">
+                                                    Subscribe to Our Newsletter
+                                                </span>
+                                            </a>
+                                        </div>
+                                        <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                            <a class="cmp-button" type="button">
+                                                <span class="cmp-button__text">
+                                                    Shop LeaderImpact Merch                                                </span>
+                                            </a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="container responsivegrid social aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                                <div class="cmp-container">
+                                    <div class="social-title">
```

Comment:
@Prystal254 If you need a custom class, it needs to be on the component root element. The`<div>` here needs to have a `.title` class.